### PR TITLE
[framework] refactored SelectToggle component

### PR DESCRIPTION
**Pull request by pesektomas**
| Q             | A
| ------------- | ---
|Description, reason for the PR| SelectToggle component was not working with select2 library.
|New feature| No 
|[BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/)| No 
|Fixes issues| ... 
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: upgrade/UPGRADE-v9.0.0-dev.md [hunk 1]
```diff
@@ -1381,6 +1381,9 @@ There you can find links to upgrade notes for other versions too.
 - fix symfony `dump()` function ([#1745](https://github.com/shopsys/shopsys/pull/1745))
     - see #project-base-diff to update your project
 
+- refactored `SelectToggle` component ([#1803](https://github.com/shopsys/shopsys/pull/1803))
+    the js class `ToggleOption` from framework node modules was removed
```
Comment (TomasLudvik) [inline upgrade/UPGRADE-v9.0.0-dev.md:1385]: Upgrade notes should be written in manner of orders people should do, so it should be like:
`ToggleOption` js class has been removed, update your code appropriately

**Comment by TomasLudvik** [comment]
As written in Trello, while editing customer on second domain I get price groups for first domain in select form type.

File: upgrade/UPGRADE-v9.0.0-dev.md (hunk 1 above)
Comment (pesektomas) [reply upgrade/UPGRADE-v9.0.0-dev.md]: 6e89aa0ef9dfcbcb9408ef4d441d1aecdc82a5c1

**Comment by pesektomas** [comment]
> As written in Trello, while editing customer on second domain I get price groups for first domain in select form type.

7c2a7930693261b28e721c3c37bb386bb34e0bdb

**Pull request merged**